doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -284,155 +294,81 @@ Induction
-   inductive.
-
-.. tacv:: induction @term using @term with @bindings
-
-   This behaves as :tacn:`induction … using …` but also providing instances
-   for the premises of the type of the second :n:`@term`.
-
-.. tacv:: induction {+, @term} using @qualid
-
-   This syntax is used for the case :n:`@qualid` denotes an induction principle
-   with complex predicates as the induction principles generated by
-   ``Function`` or ``Functional Scheme`` may be.
-
-.. tacv:: induction @term in @goal_occurrences
-
-   This syntax is used for selecting which occurrences of :n:`@term` the
-   induction has to be carried on. The :n:`in @goal_occurrences` clause is an
-   occurrence clause whose syntax and behavior is described in
-   :ref:`occurrences sets <occurrenceclauses>`. If variables or hypotheses not
-   mentioning :n:`@term` in their type are listed in :n:`@goal_occurrences`,
-   those are generalized as well in the statement to prove.
-
-   .. example::
-
-      .. coqtop:: reset all
-
-         Lemma comm x y : x + y = y + x.
-         induction y in x |-   *.
-         Show 2.
+.. tacn:: elim @one_term_with_bindings {? using @one_term {? with @bindings } }
 
-.. tacv:: induction @term with @bindings as @or_and_intropattern using @term with @bindings in @goal_occurrences
-          einduction @term with @bindings as @or_and_intropattern using @term with @bindings in @goal_occurrences
+   An older, more basic induction tactic.  We recommend using :tacn:`induction`
+   instead where possible.  :n:`@one_term` must have an inductive
+   type.  The tactic applies the appropriate :term:`induction principle` based
+   on the type of the goal.  (For example, if the local context
+   contains :g:`n : nat` and the current goal is :g:`T` of type :g:`Prop`, then
+   :n:`elim k` is equivalent to :tacn:`apply` :n:`nat_ind with (n:=k)` since
+   `nat_ind` is for `Prop` goals.)
+   ``elim`` only modifies the goal; it does not modify the :term:`local context`.
 
-   These are the most general forms of :tacn:`induction` and :tacn:`einduction`. It combines the
-   effects of the with, as, using, and in clauses.
-
-.. tacv:: elim @term
-   :name: elim
-
-   This is a more basic induction tactic. Again, the type of the argument
-   :n:`@term` must be an inductive type. Then, according to the type of the
-   goal, the tactic ``elim`` chooses the appropriate destructor and applies it
-   as the tactic :tacn:`apply` would do. For instance, if the local context
-   contains :g:`n:nat` and the current goal is :g:`T` of type :g:`Prop`, then
-   :n:`elim n` is equivalent to :n:`apply nat_ind with (n:=n)`. The tactic
-   ``elim`` does not modify the context of the goal, neither introduces the
-   induction loading into the context of hypotheses. More generally,
-   :n:`elim @term` also works when the type of :n:`@term` is a statement
+   More generally,
+   :n:`elim @one_term` also works when the type of :n:`@one_term` is a statement
```

Comment:
This paragraph also entirely applies to `induction` (and was forgotten from its documentation).

Reply:
Forgotten?  Did I ever know in the first place :-)?

I copied the paragraph, but I think it overlaps a bit with the text in the preceeding paragraph: "and induction hypotheses are added to the local context using names whose prefix is **IH**"

Reply:
>Forgotten? Did I ever know in the first place :-)?

Forgotten but not by you. Before this PR.